src/components/Request/FormFields/OrganizationalPOC.tsx

```diff
@@ -0,0 +1,52 @@
+import { Label, Text } from "@fluentui/react-components";
+import { ContactIcon } from "@fluentui/react-icons-mdl2";
+import { Controller } from "react-hook-form";
+import "components/Request/Request.css";
+import { FormField } from "components/Request/NewRequestForm";
+import { PeoplePicker } from "components/PeoplePicker/PeoplePicker";
+
+const OrganizationalPOC = ({ name, form }: FormField) => {
+  return (
+    <div className="requestFieldContainer">
+      <Label
+        id={name + "Id"}
+        size="small"
+        weight="semibold"
+        className="requestFieldLabel"
```

Comment:
Add required attribute to label
```suggestion
        className="requestFieldLabel"
        required
```